### DISCUSSION TITLE
New feed is great

### DISCUSSION THREAD

**Discussion by TodePond** [community/community#67179] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

The new feed is great, I don't know what everyone is complaining about. I love it, nice job!

**Comment by leeoniya** [comment]
>  I don't know what everyone is complaining about

you would know if you actually *read* any of the thousand complaints 🤷

**Reply by TodePond** [reply]
No really, I like it! I'm happy to explain how it has helped me, if that helps

**Reply by leeoniya** [reply]
yes, i think it would be good to understand what you like about it more than the old feed.

**Reply by loglot** [reply]
i mean, they added it to dream berd because they liked it so much

**Reply by TodePond** [reply, marked as answer]
Sure thing! The new feed has introduced me to some wonderful new projects. For example, I've now discovered loads of github alternatives like gitlab, sourcehut, youtrack, and many more. I'm currently moving all of my projects over to them!

**Reply by leeoniya** [reply]
i agree, i don't think i would have heard of https://codeberg.org/ or forgejo without this debacle. Gitlab's UI never resonated with me, and i might just self-host on Linode at some point soon.

maybe GH won't mind 15k stars-worth of repos moving off their platform overnight :man_shrugging: . at least they'll have their algorithmic feed for comfort.

**Reply by TodePond** [reply]
Thanks that's useful to know, I'll try those out!

**Comment by stuartclayman** [comment]
It's not great.  I don't want such nonsense in a tool.

People can go to Farcebook or linkedin or other places if you want feeds.

Can someone tell me how to turn this off.

**Comment by ghost** [comment]
it was just fixed recently. one or two days ago, it was awful because the "stars" and similar options were broken. nice troll.

**Reply by TodePond** [reply]
Yes you're right, it had some issues to start with, but now it's great!

**Comment by Alireza199666** [comment]
like project

**Comment by todirbg** [comment]
> ### Select Topic Area
> 
> Product Feedback
> ### Body
> 
> The new feed is great, I don't know what everyone is complaining about. I love it, nice job!

Great trolling dude 🥇 👍

**Comment by SatanKolla666** [comment]
Respect #2
